functional-test/pom.xml

```diff
@@ -36,6 +36,9 @@
     <ds.connection.url>jdbc:mysql://localhost:${mysql.port}/${ds.database}?characterEncoding=UTF-8</ds.connection.url>
     <ds.driver.class>com.mysql.jdbc.Driver</ds.driver.class>
 
+    <mysql.dist.version>5.5.34</mysql.dist.version>
+    <!-- <mysql.dist.version>5.6.21</mysql.dist.version> -->
```

Comment:
I'm trying to test against both versions, and for the moment it's easier than switching the build params in jenkins.